rebalance/rebalance.py

```diff
@@ -684,6 +684,37 @@ def health_score(liquidity):
     return score * coefficient
 
 
+def get_avg_forward_fees(plugin: Plugin, intervals):
+    now = time.time()
+    max_interval = max(intervals)
+    total = [0] * len(intervals)
+    fees = [0] * len(intervals)
+    res = [0] * len(intervals)
+    # TODO: next commented out line dows not work. pyln binding bug?
+    #all_forwards = plugin.rpc.listforwards(status="settled")["forwards"]
+    all_forwards = list(filter(lambda fwd: fwd.get("status") == "settled"
+                                  and fwd.get("resolved_time", 0) + max_interval * 60 * 60 * 24 > now,
+                               plugin.rpc.listforwards()["forwards"]))
+
+    # sort by age, newest first
+    all_forwards.sort(key=lambda fwd: fwd["resolved_time"], reverse=True)
+
+    # build intermediate result per interval
+    for fwd in all_forwards:
+        if now > fwd["resolved_time"] + max_interval * 60 * 60 * 24:
+            break
```

Comment:
No need for this, because we filter out older forwards.

Reply:
Right, now thats pointless